examples/model_selection/plot_tuned_decision_threshold.py

```diff
@@ -70,29 +70,46 @@
 # repetitions of 5-fold cross-validation.
 import pandas as pd
 
+from sklearn.metrics import make_scorer, recall_score
 from sklearn.model_selection import RepeatedStratifiedKFold, cross_validate
 
-scoring = ["accuracy", "balanced_accuracy"]
+scoring = {
+    "accuracy": "accuracy",
+    "balanced_accuracy": "balanced_accuracy",
+    "recall": make_scorer(recall_score, pos_label=pos_label),
```

Comment:
I did not read the rest of the example but if we want to optimize a metric under constraint, let's use `precision` and `recall`. The idea is to maximize the recall given a certain level of `precision`. So we could compute these metric.

I have to check the entire example but we could drop the accuracy and balanced accuracy then.